src/app/containers/StakePage/components/CurrentStakes.tsx

```diff
@@ -195,7 +194,7 @@ function AssetRow(props: AssetProps) {
         </div>
       </td>
       <td className="tw-text-left tw-font-normal">
-        {weiToNumberFormat(props.item[0])} {t(translations.stake.sov)}
+        {numberFromWei(props.item[0])} {t(translations.stake.sov)}
```

Comment:
I don't think we need this level of precision. I would use `weiToFixed(props.item[0], 3)`since we display only 3 decimals in `Total Staked SOV`.

![image](https://user-images.githubusercontent.com/14878713/125284930-5767b000-e31a-11eb-82c2-41d6a8c47020.png)